src/fu-engine.c

```diff
@@ -89,6 +89,7 @@
 #endif
 
 #define MINIMUM_BATTERY_PERCENTAGE_FALLBACK 10
+#define INOTIFY_MIN_USER_INSTANCES	    64
```

Comment:
Is this really going to find anything? 64 is veryyyyy low. Given the commit linked I have an idea.

How about if you detect it's less than 1% of system ram just leave a warning in the logs.

Reply:
>64 is veryyyyy low

Any higher than that and my fwupd started up just fine. I'm hesitant to make this a percentage of the RAM as that gets into all kind of edge cases. How about we make it hardcoded to 8192/2, i.e. 4096?

Reply:
>Any higher than that and my fwupd started up just fine

Right - this should only be an issue while system is under pressure.  Like if you started up vscode with the C++ extension and opened a kernel directory and then started fwupd.

>How about we make it hardcoded to 8192/2, i.e. 4096

I'm not sure I agree with this either though.  It's not going to be a deterministic failure.  What about instead moving it back to the failure path like my original idea?

Reply:
Like https://github.com/fwupd/fwupd/pull/4060 ?

Reply:
Yup, I think that's a lot more useful.